### PR TITLE
Changes from tord comments (#14)

### PR DESCRIPTION
**Pull request by rogermeadows**
These changes are based on excellent and much appreciated feedback from Tord.
I think all of the suggested changes have been incorporated.

### CODE REVIEW

File: git-glossary.md [hunk 1]
```diff
@@ -23,17 +28,20 @@ place where the full version historyof the software is stored (like CVS or Subve
 of the code is also a repository that can contain the full history of all changes.
 
 ## github
-`github` is a website and a service that maintains git repositories for project teams. Developers can interact with repositories
-atored in `gtihub` using it's web interface or using the `git` command line interface.
+`github` is a website and a service that maintains remote git repositories for project teams. Developers can interact with 
+these remote repositories stored in `gtihub` using the web interface. If you clone a repository to your development
```
Comment (SunyataZero) [inline git-glossary.md]: A few minor suggestions: "stored in `gtihub` using the web interface" -> "stored on github using their web interface"

File: git-glossary.md [hunk 2]
```diff
@@ -43,8 +51,8 @@ A collection of [references](#references) to commit objects and an [object datab
...
 ## references
 A string name that points to a commit. A reference also includes information about where the [commit](#commit) is located.
-For fork-clone style development the commit will be in one of three locations. 
+For the [fork/clone development workflow](fork-clone-workflow.md) the commit will be in one of three locations. 
 
 1. A local clone on a developer's computer
-1. The origin repository in the cloud
-1. The upstream repository in the cloud
+1. The origin repository in [github](#github), which is your remote [fork](#fork)
+1. The upstream repository in [github](#github), which is the organization's remote repository
```
Comment (SunyataZero) [inline git-glossary.md]: Do we want to mention the organization here? It's possible that the repo is held by an individual account right? I'm not sure what to call this though, "main repository"? Something else?

File: git-glossary.md [hunk 3]
```diff
@@ -13,6 +13,11 @@ first 7 characters of the 40-character string.
 
 As a verb: The action of storing a new snapshot of the project's state in the Git history, by creating a new commit. 
 
+## fork
+A fork is a copy of a repository. Forking a repository allows you to freely experiment with changes without
```
Comment (SunyataZero) [inline git-glossary.md:17]: Can we add here that a fork is stored on github and that it maintains a connection to the original repo? I've spent 30 minutes reading about the difference between clones and forks, trying to wrap my head around it. The connection to the original repo and that they are stored on github seem to stick out to me, but i'm a newbie in this area

**Review by SunyataZero — CHANGES_REQUESTED** [review]

File: git-glossary.md (hunk 1 above)
Comment (rogermeadows) [reply git-glossary.md]: Agreed. Made the change.

File: git-glossary.md (hunk 2 above)
Comment (rogermeadows) [reply git-glossary.md]: Yes. I sometimes write the definition in git-glossary as if it was in the work flow section where we would want to use the term organization. Here, I'll change it to 'main repository'

Comment (rogermeadows) [reply git-glossary.md]: Or 'main remote repository'.

File: git-glossary.md (hunk 3 above)
Comment (rogermeadows) [reply git-glossary.md:17]: I amended the definition of fork. Please see if it is what you had in mind.

**Review by SunyataZero — APPROVED** [review]

**Pull request merged**